src/content/changes/highlights/2023-07-27.md

```diff
@@ -0,0 +1,25 @@
+---
+date: 2023-07-27T14:00:00
+title: Highlights for the week ending July 27 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Nginx Ingress Controller 
+We have spent the last months on reworking our [nginx-ingress-controller-app](https://github.com/giantswarm/ingress-nginx-app) by aligning it to the [upstream ingress-nginx]u(https://github.com/kubernetes/ingress-nginx) project. This was necessary to be future proof, feature compliant and to offer the best ingress experience possible. Therefor we want to announce the first public beta release of our new `ingress-nginx chart`. At the moment it is not recommended to install this release in a production environment, it is for testing purposes only. The stable release v3.0.0 will be cut as soon as all possible bugs have been fixed and improvements have been implemented. For our  [beta release](https://github.com/giantswarm/ingress-nginx-app/releases/tag/v3.0.0-beta1) we added all changes mentioned in our [alpha1 release](https://github.com/giantswarm/ingress-nginx-app/releases/tag/v3.0.0-alpha1).
```

Comment:
```suggestion
## Nginx Ingress Controller 

We have spent the last months reworking our [nginx-ingress-controller-app](https://github.com/giantswarm/ingress-nginx-app) by aligning it to the [upstream ingress-nginx]u(https://github.com/kubernetes/ingress-nginx) project. This is necessary to be future-proof, feature compliant, and offer the best ingress experience possible. Therefore we want to announce the first public beta release of our new `ingress-nginx chart`.

The stable release v3.0.0 will be cut as soon as all possible bugs have been fixed and improvements have been implemented. For our  [beta release](https://github.com/giantswarm/ingress-nginx-app/releases/tag/v3.0.0-beta1) we added all changes mentioned in our [alpha1 release](https://github.com/giantswarm/ingress-nginx-app/releases/tag/v3.0.0-alpha1).
```
I removed the warning since it is now below